### PR TITLE
Update de.json

### PR DESCRIPTION
**Pull request by sschuste**

### CODE REVIEW

File: ui/src/i18n/de.json
Comment (TautFlorian) [inline ui/src/i18n/de.json:110]: There is a misspelling. Should be `Logdatei abschneiden`

File: ui/src/i18n/de.json
Comment (TautFlorian) [inline ui/src/i18n/de.json:106]: `or` should be translated as well to `oder` maybe `und/oder` would fit even better.

File: ui/src/i18n/de.json
Comment (TautFlorian) [inline ui/src/i18n/de.json:67]: `überschreiben` should be `überschrieben`

File: ui/src/i18n/de.json
Comment (TautFlorian) [inline ui/src/i18n/de.json:209]: Should be `aus dem Cache`

File: ui/src/i18n/de.json
Comment (TautFlorian) [inline ui/src/i18n/de.json:220]: Should be `Entkopple`

File: ui/src/i18n/de.json
Comment (TautFlorian) [inline ui/src/i18n/de.json:239]: In my opinion this translation isn't better than the one before.
I would not translate it this way.

**Review by TautFlorian — CHANGES_REQUESTED** [review]
There are some misspellings and my opinions about some of the changes you made. 
Would appreciate to hear your thoughts on this.

File: ui/src/i18n/de.json
Comment (TautFlorian) [inline ui/src/i18n/de.json:107]: Maybe we should talk generally about dots. In this case there is a dot added to the line but in most cases there is none.
We should decide for one option, in my eyes we shouldn't add a line ending point.

**Review by TautFlorian — CHANGES_REQUESTED** [review]

**Comment by oznu** [comment]
@sschuste - I'll wait for your comment on the above before merging 😄 

Thanks everyone for contributing!

**Comment by TautFlorian** [comment]
@oznu  I'm going to create a pull request by my own fixing `de.json` this evening because @sschuste seems not to be very active.

**Comment by oznu** [comment]
I'll merge this. @TautFlorian - feel free to do another PR with any changes you think improve the translation further 😄 

Thanks @sschuste

**Pull request merged**
